examples/rootca.key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This also shouldn't be committed.

Reply:
Fixed, and updated gen-certs.sh to follow this policy.